sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleService.java

```diff
@@ -181,11 +181,11 @@ public List<SampleIndexDto> getIndexList(SampleCriteria sampleCriteria, Integer
 		SampleJoins joins = sampleQueryContext.getJoins();
 
 		final Join<Sample, Case> caze = joins.getCaze();
-		final Join<Case, District> caseDistrict = joins.getCaseDistrict();
+		final Join<Case, District> caseDistrict = joins.getCaseResponsibleDistrict();
```

Comment:
should be a logic here:

district if it's set, responsible district otherwise

Reply:
updated